### PR TITLE
Add results page for surveys

### PR DESCRIPTION
**Pull request by kevinhquach**
Has some primitive histogram display that will be deprecated with proper js libraries

### CODE REVIEW

File: src/main/java/com/based/basedsurvey/controller/FillSurveyController.java [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+public class FillSurveyController {
+
+    private SurveyRepository surveyRepository;
+
+    @Autowired
+    public FillSurveyController(SurveyRepository surveyRepository) {
+        this.surveyRepository = surveyRepository;
+    }
+
+    @GetMapping("/survey/{surveyID}/answer")
```
Comment (Shashaank5) [inline src/main/java/com/based/basedsurvey/controller/FillSurveyController.java:25]: Does the curly braces mean the parameter is optional or is it a variable. I'm guessing it's the latter and in that case looks good

File: src/main/java/com/based/basedsurvey/controller/FillSurveyController.java [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+        model.addAttribute("surveyName", s.getName());
+
+        // if survey is closed
+        if (!s.isOpen()) {
+            model.addAttribute("issue", "Survey is closed");
+            return "SurveyFillIssue";
+        }
+
+        // if survey has no questions
+        if (s.getQuestions().size() == 0) {
```
Comment (Shashaank5) [inline src/main/java/com/based/basedsurvey/controller/FillSurveyController.java:44]: Will issue be overwritten if it's closed and size is 0 and if so we may need to account for that in the future

File: src/main/java/com/based/basedsurvey/controller/ResultsSurveyController.java [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+    /**
+     * Given an OpenAnswerQuestion, return a String that represents all of its responses in html.
+     * @param q OpenAnswerQuestion
+     * @return String of html elements
+     */
+    private String getOpenAnswerResults(OpenAnswerQuestion q) {
+        String s = "";
+        //doesn't quite work, but this should probably be done with css
+        s += "<h3>" + q.getPrompt() + "</h3><div height='200px' overflow='scroll'>"; // could specify a class/id for the div
```
Comment (Shashaank5) [inline src/main/java/com/based/basedsurvey/controller/ResultsSurveyController.java:100]: nit: you can add css in the static folder under resoures

Comment (Shashaank5) [reply src/main/java/com/based/basedsurvey/controller/ResultsSurveyController.java:100]: the index page already does this so you can look there

**Review by Shashaank5 — APPROVED** [review]
Looks overall good to me, few nitpicks but I think we can fix them later and approve for now

**Review by DavidHaighton — APPROVED** [review]

**Pull request merged**